tests/performance_tests/performance_test_basic.py

```diff
@@ -557,6 +556,18 @@ def createBaseArgumentParser():
         ptbBaseParserGroup.add_argument("--contract-dir", type=str, help="Path to contract dir", default="unittests/contracts/eosio.system")
         ptbBaseParserGroup.add_argument("--wasm-file", type=str, help="WASM file name for contract", default="eosio.system.wasm")
         ptbBaseParserGroup.add_argument("--abi-file", type=str, help="ABI file name for contract", default="eosio.system.abi")
+        ptbBaseParserGroup.add_argument("--wasm-runtime", type=str, help="Override default WASM runtime (\"eos-vm-jit\", \"eos-vm\")\
+                                         \"eos-vm-jit\" : A WebAssembly runtime that compiles WebAssembly code to native x86 code prior to\
+                                         execution. \"eos-vm\" : A WebAssembly interpreter.", default="eos-vm-jit")
```

Comment:
This should probably make use of `choices`.  See `--cluster-log-lvl` or `--database-map-mode` for examples.